admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+# Extract metadata information from documents
+
+## Overview
+
+### Introduction
+
+The metadata extraction tool is designed to help you quickly identify, retrieve, and save key details from PDF documents, such as titles, dates, keywords, and other essential information. Powered by machine learning, this tool automates much of the extraction process, saving you time and making it easier to analyze large sets of documents. Whether you’re documenting case studies, managing witness statements, or reviewing legal reports, this tool simplifies data extraction, allowing you to focus on analysis and decision-making.
```

Comment:
Hi Alex! Thanks for adding me as a reviewer. I'll start to share some considerations and questions in the comments, but please feel free to disregard them as they are merely the product of my contemplations most of the times.

`Whether you document case studies, manage witness statements, or review legal reports, this tool simplifies data extraction, allowing you to focus on analysis and decision-making`. 

I am not sure about decision-making here, maybe, advocacy or legal efforts would fit here better?

Reply:
Resolved with [3c67ca6](https://github.com/huridocs/uwazi-documentation/pull/49/commits/3c67ca67ffee57322777d3a357d5fa4194de4551)